frontend/src/lib/user.ts

```diff
@@ -26,6 +26,14 @@ type RegisterResponseErrors = {
 
 type ApiLexboxAudience = 'LexboxApi' | 'Unknown';
 
+export enum FeatureFlag {
+  FwLiteBeta = 'FW_LITE_BETA',
+}
+
+export const allPossibleFlags: FeatureFlag[] = [
```

Comment:
rather than hard coding the list which might get out of date I think you can get all the values of a ts enum, I'd look into that

Reply:
Also, GQL will presumably have generated a FeatureFlag enum for you.
It will likely be serialized differently than is done for the token, but so far we've mostly been able to map them somehow so we can avoid making a new one by hand.
I see now that I didn't do that for the LexboxAudience enum.

You could, for example, map them by removing non letters and then doing a case-insensitive compare.
We have this for example:
`const role = Object.values(UserRole).find(r => r.toLowerCase() === jwtRole) ?? UserRole.User;` which is simpler, because roles are always a single word.

Reply:
Got this figured out in commit 1a73ef4b. Thanks for the hint about `Object.values(EnumType)`, that would have taken me a little time to find.